### PR TITLE
MVP & stretch

### PR DESCRIPTION
**Pull request by justinelai**

### CODE REVIEW

**Comment by justinelai** [comment]
Ready for review. Thank you!

File: js/index.js [hunk 1]
```diff
@@ -40,3 +40,97 @@ const siteContent = {
...
+const topAboutText = document.querySelector(".top-content .text-content:nth-of-type(2) p")
+topAboutText.textContent = siteContent["main-content"]["about-content"]
+
+
+// Main > Middle
+
+const middleImg = document.getElementById("middle-img");
+middleImg.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
+// Main > Bottom
```
Comment (chrisipanaque) [inline js/index.js]: great use of comments

File: js/index.js [hunk 2]
```diff
@@ -40,3 +40,97 @@ const siteContent = {
...
+});
+
+const breakingNewsLink = document.createElement("a");
+const linkText = document.createTextNode("Breaking News!");
+breakingNewsLink.appendChild(linkText);
+breakingNewsLink.style.fontStyle = "italic";
+document.querySelector("nav").prepend(breakingNewsLink);
+
+const saleLink = document.createElement("a");
+const saleLinkText = document.createTextNode("SALE");
```
Comment (chrisipanaque) [inline js/index.js]: Great use of declarative variable names.

File: js/index.js [hunk 3]
```diff
@@ -40,3 +40,97 @@ const siteContent = {
...
+ctaH1.innerHTML = (siteContent["cta"]["h1"]).split(" ").join('<br />');
+
+ctaH1.style.textShadow = "2px 2px 5px cyan"
+
+const ctaButton = document.querySelector(".cta-text button");
+ctaButton.textContent = siteContent["cta"]["button"];
+
+const ctaImg = document.getElementById("cta-img");
+ctaImg.setAttribute('src', siteContent["cta"]["img-src"]);
+
```
Comment (chrisipanaque) [inline js/index.js]: Outstanding code organization.

File: js/index.js [hunk 4]
```diff
@@ -40,3 +40,97 @@ const siteContent = {
...
+// Contact
+
+const contact = document.querySelector(".contact h4")
+contact.textContent = siteContent["contact"]["contact-h4"]
+
+const contactPara = document.querySelectorAll(".contact p")
+contactPara[0].textContent = siteContent["contact"]["address"]
+contactPara[1].textContent = siteContent["contact"]["phone"]
+contactPara[2].textContent = siteContent["contact"]["email"];
+
```
Comment (chrisipanaque) [inline js/index.js]: Great use of line space to provide easy to read code for other engineers.

**Pull request merged**